pytac/units.py

```diff
@@ -33,6 +33,7 @@ class UnitConv(object):
     **Attributes:**
 
     Attributes:
+        id (int): The unit conversion id as it appears in the csv files.
```

Comment:
`name` has now replaced `id` so this line in the attributes sections of all the unit convs should become `name (str): An identifier for the unit conversion object.`?